### ISSUE TITLE
After updating to 2.9.7, Quick Rename does nothing anymore (temporary fluke?)

### ISSUE THREAD

**Issue by randomname42** [TV-Rename/tvrename#654]
Edit: Closing and reopening TV Rename seems to have fixed this. But the problem was really strange, so I'll leave the issue here in case anyone else has it happen.

## Expected Behaviour

Dragging files to be renamed onto Quick Rename used to allow you to rename.

## Current Behaviour, Failure Information, and Steps to Reproduce

1. I opened TV Rename after updating.
2. First thing I did was open Quick Rename.
3. Then I dragged a folder onto Quick Rename.
4. All that happens is there's a "not allowed" symbol (black circle with a diagonal line). Nothing else happens at all.
5. Dragging either a folder or a file onto the Quick Rename area causes that not allowed symbol, and nothing else happens.

This was working fine before 2.9.7. I don't have a clue why this is going on now.

Doesn't matter which kind of video file, what show, or folder name. Nothing works at all.

Having the Log Viewer open doesn't show anything happening either.

Hitting View Full Log showed that nothing at all was happening when dragging files/folders onto Quick Rename.

## Context

Using Windows, TV Rename version 2.9.7

Edit: This happened in Quick Rename with Show on \<Auto\>, and still happened even after selecting the correct show manually.

The Format box is greyed out and can't be clicked. I don't remember if it's usually like that, as I haven't ever had to mess with Format when using Quick Rename.

Edit 2: These steps did allow Quick Rename to work:

1. Closed and reopened TV Rename.
2. Instead of opening Quick Rename right away, I decided to try switching to the Scan tab first.
3. Then I opened Quick Rename.
4. Now dragging onto Quick Rename worked like it should.

After closing and reopening TV Rename again, Quick Rename is still works right. Even if opening Quick Rename is the first thing I do.

I really don't understand what could have caused the problem in the first place, or why closing and reopening TV Rename would permanently fix it.

**Comment by randomname42** [comment]
Here's everything the log said, in case it's any use:

> 2019-11-29 17:07:19|FATAL| TV Rename 2.9.7 logging started on Microsoft Windows NT 6.2.9200.0, 64 Bit OS, 64 Bit Process 4.0.30319.42000 Interactive with args:  
> 2019-11-29 17:07:19|INFO| Copyright (C) 2019 TV Rename 
> 2019-11-29 17:07:19|INFO| This program comes with ABSOLUTELY NO WARRANTY; This is free software, and you are welcome to redistribute it under certain conditions 
> 2019-11-29 17:07:20|INFO| Splash Screen Updated with: Filling Shows 
> 2019-11-29 17:07:20|INFO| Splash Screen Updated with: Updating WTW 
> 2019-11-29 17:07:20|INFO| Splash Screen Updated with: Write Upcoming 
> 2019-11-29 17:07:20|INFO| Splash Screen Updated with: Write Recent 
> 2019-11-29 17:07:20|INFO| Splash Screen Updated with: Setting Notifications 
> 2019-11-29 17:07:20|INFO| Splash Screen Updated with: Creating Monitors 
> 2019-11-29 17:07:20|INFO| Splash Screen Updated with: Starting Monitor 
> 2019-11-29 17:07:20|INFO| Splash Screen Updated with: Running autoscan

**Comment by Numbergod** [comment]
I'm seeing similar weirdness since the update to 2.9.8  Quick scan often doesn't work or show next airing (unless run straight after a Recent scan).  And any scan now takes ten times as long as it previously does, as it seems to be downloading a heap more info.

I've performed a couple of Force Refresh All's since updating.  I've only noticed the behaviour since, but it may have been occurring since the upgrade to 2.9.8.

I don't think I've yet gathered enough information on the behaviour for a full bugreport.  I may look into that if I get a little time to play...
